<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>9999999</width>
     <height>9999999</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(15, 91, 255);
	border-radius: 7px;
	color:white
}

QPushButton::Pressed
{
background-color: rgb(74, 87, 94);
border-radius: 7px;
color:white
}

QPushButton::Released
{
	background-color: rgb(15, 91, 255);
	border-radius: 7px;	
	color:white
}</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordAgainEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>330</y>
     <width>550</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>608</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(44, 56, 68);
	border-style: solid;
	border-color: rgb(74, 87, 94);
	border-width: 2;
	border-radius: 5px;
	color: rgb(126,132,138)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Повторить пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>270</y>
     <width>550</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>608</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(44, 56, 68);
	border-style: solid;
	border-color: rgb(74, 87, 94);
	border-width: 2;
	border-radius: 5px;
	color: rgb(126,132,138)
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>210</y>
     <width>550</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(44, 56, 68);
	border-style: solid;
	border-color: rgb(74, 87, 94);
	border-width: 2;
	border-radius: 5px;
	color: rgb(126,132,138)
}

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Имя пользователя</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>600</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="messageLable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>480</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>608</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>50</y>
     <width>510</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="authorizationButton">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(74, 87, 94);
border-radius: 0px;
color:white
}</string>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registrationButton">
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(15, 91, 255);
border-radius: 0px;
color: white;
}</string>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ButtonImage" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonImage</class>
   <extends>QPushButton</extends>
   <header>Other/buttonimage.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>loginEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>passwordAgainEdit</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>authorizationButton</tabstop>
  <tabstop>registrationButton</tabstop>
  <tabstop>settingsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
